library/globals.inc.php

```diff
@@ -938,13 +938,55 @@
       xl('This will use the custom Statment showing the description instead of the codes.')
     ),
 	 
+    'show_aging_on_custom_statement' => array(
+      xl('Show Aging on Custom Statement'),
+      'bool',                           // data type
+      '0',                              // default = false
+      xl('This will Show Aging on the custom Statment.')
+    ),
+
+    'use_statement_print_exclusion' => array(
+      xl('Exclude Statements from Printing'),
+      'bool',                           // data type
+      '0',                              // default = false
+      xl('This will use the Following settings to Exclude Patient Statements from Printing.')
```

Comment:
Will there be more than 1 exclusion setting in the future, or will the minimum_amount_to_print be the only one?